lms/static/sass/discussion/_discussion.scss

```diff
@@ -808,35 +499,13 @@ body.discussion {
               color: #888;
             }
 
+            // deprecated styles - remove after review / cleanup
             header {
-              padding-bottom: 0;
```

Comment:
Context:  These styles were also moved to _thread.scss. They are shared across posts, responses, and comments as well. (+thread - elements - shared styles)